pjp.tm11/src/main/webapp/login/js/main.js

```diff
@@ -0,0 +1,70 @@
+
+(function ($) {
+    "use strict";
+
+
+    /*==================================================================
+    [ Focus Contact2 ]*/
+    $('.input100').each(function(){
+        $(this).on('blur', function(){
+            if($(this).val().trim() != "") {
+                $(this).addClass('has-val');
+            }
+            else {
+                $(this).removeClass('has-val');
+            }
+        })    
+    })
+  
+  
+    /*==================================================================
+    [ Validate ]*/
+    var input = $('.validate-input .input100');
+
+    $('.validate-form').on('submit',function(){
+        var check = true;
+
+        for(var i=0; i<input.length; i++) {
+            if(validate(input[i]) == false){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/ohbus/PJP/pullRequest?prid=5652670)